### DISCUSSION TITLE
3D_UNIFORM_COLOR

### DISCUSSION THREAD

**Discussion by emreurall** [carson-katri/dream-textures#759] [Q&A]
Hey,
im using blender 4.0 and installed dream texture 0.3.1 
installed model "stabilityai/stable-diffusion-2-depth"
settings up like that
![DreamTexture](https://github.com/carson-katri/dream-textures/assets/75078829/637807ae-4e9d-4ffd-a678-fb9eaac92808)

When I click on the "Project Dream Texture" button, I get the following error

**Python: Traceback (most recent call last):
  File "C:\Users\...\AppData\Roaming\Blender Foundation\Blender\4.0\scripts\addons\dream_textures\engine\annotations\depth.py", line 32, in _execute
    shader = gpu.shader.from_builtin('3D_UNIFORM_COLOR')
ValueError: expected a string in ('FLAT_COLOR', 'IMAGE', 'IMAGE_COLOR', 'SMOOTH_COLOR', 'UNIFORM_COLOR', 'POLYLINE_FLAT_COLOR', 'POLYLINE_SMOOTH_COLOR', 'POLYLINE_UNIFORM_COLOR'), got '3D_UNIFORM_COLOR'**

and the blender gets stuck.

**Comment by yoyoyeahyoyo** [comment]
i found ez fix open that file where the error is at that line:
`File "C:\Users...\AppData\Roaming\Blender Foundation\Blender\4.0\scripts\addons\dream_textures\engine\annotations\depth.py", line 32, in _execute`
And change 3D_UNIFORM_COLOR to UNIFORM_COLOR and it ran fine

if you look at what these constants are in the docs for latest:
https://docs.blender.org/api/current/gpu.shader.html
vs. older:
https://docs.blender.org/api/2.91/gpu.shader.html
You can see the change in constants

Once i changed that line it ran fine, i was usin linux
